Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -24,11 +27,11 @@ public final class ProjectPamphletViewController: UIViewController {
   }
 
   public override var prefersStatusBarHidden: Bool {
-    return self.viewModel.outputs.prefersStatusBarHidden
+    return UIApplication.shared.statusBarOrientation.isLandscape
   }
 
-  public override var preferredStatusBarUpdateAnimation: UIStatusBarAnimation {
-    return .fade
+  public override var preferredStatusBarStyle: UIStatusBarStyle {
```

Comment:
How does this change affect iOS 10?

Reply:
It should work fine for both iOS 10 and 11.